Update migrate-to-exchange-online.md (#2753)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md

- We recommend that you don't use Outlook's PST export feature to migrate public folders to Microsoft 365, Office 365, or Exchange Online. Public folder mailbox growth in Exchange Online is managed using an auto-split feature that splits the public folder mailbox when it exceeds size quotas. Auto-split can't handle the sudden growth of public folder mailboxes when you use PST export to migrate your public folders, and you may have to wait for up to two weeks for auto-split to move the data from the primary mailbox. We recommend that instead you use the cmdlet-based instructions in this article to migrate your public folders. If you still decide to migrate public folders using PST export, see [Migrate Public Folders to Office 365 by using Outlook PST export](#migrate-public-folders-to-microsoft-365-or-office-365-by-using-outlook-pst-export)  later in this article.

> - Please verify if the DefaultPublicFolderAgeLimit is configured on the organization level (Get-OrganizationConfig | fl DefaultPublicFolderAgeLimit) or if you have any AgeLimit (Get-PublicFolder <FolderPath> | fl AgeLimit) configured for the individual Public Folders, so that automatic deletions of the content to be prevented.

Reviewer: ```suggestion
- Verify if the **DefaultPublicFolderAgeLimit** parameter is configured on the organization level (Get-OrganizationConfig | fl DefaultPublicFolderAgeLimit) or if you have any age limit (Get-PublicFolder <FolderPath> | fl AgeLimit) configured for the individual public folders, to prevent automatic deletions of the content.
```
